### DISCUSSION TITLE
[Discussion] Add others voice models

### DISCUSSION THREAD

**Discussion by Brodvd** [intel/openvino-plugins-ai-audacity#331] [Ideas]
Hey @RyanMetcalfeInt8  what do you think about adding text-to-speech and voice-cloning model to OpenVINO? I found [Coqui.IA](https://github.com/coqui-ai/TTS), It seems the best, you can try It also [here](https://huggingface.co/spaces/coqui/xtts).
Instead, maybe you have already seen It, It was just published [here](https://github.com/ZFTurbo/Music-Source-Separation-Training) the pre-trained model MelBand Roformer Vocals DeReverb/DeEcho that, seeing the [examples](https://huggingface.co/Sucial/Dereverb-Echo_Mel_Band_Roformer/tree/main/examples), seems very good as voice-dereverb.

**Comment by RyanMetcalfeInt8** [comment]
Hi @Brodvd,

Thanks for the tips! I would love to eventually add text-to-speech / voice-cloning models to this project -- It's something that I've wanted to do all along but haven't yet found the time to do.. perhaps next year.

> Instead, maybe you have already seen It, It was just published [here](https://github.com/ZFTurbo/Music-Source-Separation-Training) the pre-trained model MelBand Roformer Vocals DeReverb/DeEcho that, seeing the [examples](https://huggingface.co/Sucial/Dereverb-Echo_Mel_Band_Roformer/tree/main/examples), seems very good as voice-dereverb.

Yes, I absolutely have seen that -- and am currently in the process of converting / optimizing it for use in this project. I think DeReverb is such a useful feature that this is top of my priority list. Stay tuned.

**Comment by MarcoRavich** [comment]
Just discovered [ClearerVoice-Studio](https://github.com/modelscope/ClearerVoice-Studio#readme) by @alibabasglab & @zexupan a toolkit for _speech enhancement, speech separation, speech super-resolution, target speaker extraction, and more_.

As always you can also find more resources listed in the - currently updating - [AI-Voicing page](https://github.com/FORARTfe/HyMPS/blob/main/Audio/AI-Voicing.md#---). 

Hope that inspires !
